src/pages/ApplyMain/index.js

```diff
@@ -15,14 +15,27 @@ const ApplyMainPage = () => {
   const [innerWidth, setInnerWidth] = useState(window.innerWidth);
   const DESKTOP_WIDTH = 1199;
   const TABLET_WIDTH = 599;
+  const [clikedArray, setClickedArray] = useState([false, false, false]);
+
+  const handleClick = idx => {
+    let newArr = clikedArray;
+    if (newArr[idx] == true) {
```

Comment:
`==` 쓰시면 안됩니다 :bbadda:

```suggestion
    if (newArr[idx] === true) {
```

Reply:
죄..죄송하비다